zanata-war/src/main/java/org/zanata/adapter/OkapiFilterAdapter.java

```diff
@@ -172,6 +168,9 @@ public Resource parseDocumentFile(URI documentContent,
         RawDocument rawDoc =
                 new RawDocument(documentContent, "UTF-8",
                         net.sf.okapi.common.LocaleId.fromString("en"));
+        if (rawDoc.getTargetLocale() == null) {
+            rawDoc.setTargetLocale(net.sf.okapi.common.LocaleId.ENGLISH);
```

Comment:
@djansen-redhat yes, that sounds like a bug, or a flaw in the API (why ask for a target locale?).

But if you really need to provide a target locale here, I suggest using `und` for undetermined, which is part of the ISO-639 standard. See https://en.wikipedia.org/wiki/ISO_639-2#Special_situations